common/djangoapps/xblock_django/tests/test_deprecated_xmodule_model.py

```diff
@@ -0,0 +1,58 @@
+"""
```

Comment:
The convention in edx-platform is to test all the models in the app's `model.py` in a file `test_models.py`. Can you rename this?